base/ca/src/com/netscape/ca/CAService.java

```diff
@@ -1179,8 +1211,14 @@ Extension createSCTextension(List<String> ctResponses) {
            opaque issuer_key_hash[32];
            TBSCertificate tbs_certificate;
          } PreCert;
+    *
+    * @param response CT log server response
+    * @param tbsCert encoded TBSCert
+    * @param logPublicKey public key of log
+    * @return boolean true for verified; false for not verified
+    * @author cfu
     */
-    void verifySCT(CTResponse response, byte[] tbsCert, String logPublicKey)
+    Boolean verifySCT(CTResponse response, byte[] tbsCert, String logPublicKey)
```

Comment:
You may want to use the primitive `boolean` instead of the `Boolean` object since the former is easy on memory and is cheap.